docs/cloud/earthly-ci.md

```diff
@@ -0,0 +1,149 @@
+# Earthly CI
+
+This feature is part of the Earthly CI paid plan.
+
+{% hint style='danger' %}
+##### Important
+
+This feature is currently in **Alpha** stage. **We do not recommend using Earthly CI Alpha for mission-critical pipelines.**
+
+* The feature may break or change significantly in future versions of Earthly.
+* Give us feedback on
+  * [Slack](https://earthly.dev/slack)
+  * [GitHub issues](https://github.com/earthly/cloud-issues/issues)
+  * [Emailing support](mailto:support+ci@earthly.dev)
+{% endhint %}
+
+Earthly CI is a hosted CI service that allows you to run your Earthly builds in the cloud. Earthly CI gives teams repeatable pipelines that run exactly the same in CI as on your laptop; has an automatic and instantly available build cache that makes builds faster; and is super simple to use.
+
+Earthly CI uses Earthfiles as the build configuration language.
+
+## Benefits
+
+* **Ridiculously fast** - Earthly CI uses the same build cache and build parallelization technology as Earthly Satellites, so builds are 2-20X faster compared to a traditional CI.
+* **Super simple** - Earthfiles have a super simple, instantly recognizable syntax – like Dockerfile and Makefile had a baby.
+* **Great for Monorepos and Polyrepos** - Earthly CI is great for both monorepos and polyrepos. You can organize your build logic however makes the most sense for your project. The caching ensures that only what has changed is rebuilt.
+* **Remote build runners** - Earthly CI comes with access to Earthly Satellites. This means that you can run builds from your laptop without having to commit code to Git with every attempt.
```

Comment:
Maybe change "... you can run builds from your laptop ..." to "... you can run ad-hoc builds in CI from your laptop ..." for clarity. Or reword for clarity around builds running remotely in CI.